docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.md

```diff
@@ -4,28 +4,32 @@ description: Learn about the development, staging, and production environments o
 
 ### Developer environment (DEV) (optional)
-A Dev environment is normally used to test code that just left your development team or is actually used in the development process directly through continuous deployment.
+A Dev environment is used to test code that just left your development team or is used in the development process directly through continuous deployment. It is not designed to handle load tests and large datasets. The development environment does not provide autoscaling and cannot be increased.
 
 ### Staging (STAGE)
-This environment is normally used to host a snapshot of the Dev environment - or a version of your code that is considered stable and that should be deployed to production eventually. You should use your staging environment to do performance testing to see how your code will behave in your production environment. When demoing your shop, this environment will be used regularly.
-
-Please note that it is imperative to use the same data here (in quality and quantity) as you would in production. If you cannot use the same data, make sure to create mock-ups that are at least of the same quantity as the data your production system needs to be able to handle.  Not doing so is gambling with your go-live and first production deployments.
+This environment is used to host a snapshot of the Dev environment or a stable version of your code that is deployed to production eventually. You can use your staging environment to do performance testing to see how your code behaves in your production environment. However, staging is not of the same size as production, and a subset of data and expected traffic must be considered while running the test. When demoing your shop, this environment is used regularly. The staging environment provides limited autoscaling capabilities and cannot be further scaled up.
 
 ### Production (PROD)
 
-Your production environment should host the code that the end-user gets to interact with directly. The code should already be tested on the staging environment for stability and performance. Please do not make changes to the production environments directly unless they have been verified on your lower environments first.
+Your production environment must host the code that the end-user gets to interact with directly. The code must already be tested on the staging environment for stability and performance. 
+
+{% info_block infoBox "Note" %}
+
+Do not make changes to the production environments directly unless they have been verified on your lower [environments](/docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.html) (staging or dev) first.
```

Comment:
Why provide a link to the doc you are already reading? Just replace `lower environments` with `staging and dev environments`.